app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The current logic returns immediately when a `VaccineError` is encountered, which prevents the rest of the friends from being processed. Consider accumulating results or using a flag to ensure all friends are processed before returning a message. This will allow you to check all friends and provide a comprehensive message.